src/components/file/file.repository.ts

```diff
@@ -109,11 +109,11 @@ export class FileRepository extends CommonRepository {
       .query()
       .match([
         node('start', 'FileNode', { id }),
-        relation('out', 'parent', 'parent', ACTIVE, '*'),
+        relation('in', 'child', 'child', ACTIVE, '*'),
```

Comment:
I think the query variable should continue to be called `parent` as it is one of the _parents_ of the node.
```suggestion
        relation('in', 'parent', 'child', ACTIVE, '*'),
```